src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -0,0 +1,181 @@
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import { wrapWorkspace } from 'src/pages/workspaces/workspace/WorkspaceContainer'
+
+
+const styles = {
+  sectionTableLabel: { fontWeight: 600 }
+}
+
+// Note: this can take a while with large data inputs. Consider memoization if the page ever needs re-rendering.
+const groupCallStatuses = _.flow(
+  _.values,
+  _.flattenDepth(1),
+  _.countBy(a => collapseCromwellExecutionStatus(a.executionStatus))
+)
+
+const statusCell = ({ calls }) => {
+  const { succeeded, failed, running, submitted, ...others } = groupCallStatuses(calls)
+
+  const makeRow = (count, icon, text) => {
+    return !!count && div({ style: { display: 'flex', alignItems: 'center', marginTop: '0.25rem' } }, [
+      icon,
+      span([` ${count} ${text}`])
+    ])
+  }
+
+  return h(Fragment, [
+    makeRow(submitted, submittedIcon(), 'Submitted'),
+    makeRow(running, runningIcon(), 'Running'),
+    makeRow(succeeded, successIcon(), 'Succeeded'),
+    makeRow(failed, failedIcon(), 'Failed'),
+    _.map(([name, count]) => makeRow(count, unknownIcon(), name), _.toPairs(others))
+  ])
+}
+
+const WorkflowDashboard = _.flow(
+  Utils.forwardRefWithName('WorkflowDashboard'),
+  wrapWorkspace({
+    breadcrumbs: props => breadcrumbs.commonPaths.workspaceDashboard(props),
+    title: 'Job History', activeTab: 'job history'
+  })
+)((props, ref) => {
+  const { namespace, name, submissionId, workflowId, workspace: { workspace: { bucketName } } } = props
+
+  /*
+   * State setup
+   */
+  const [workflow, setWorkflow] = useState({})
+  const [showLog, setShowLog] = useState(false)
+
+  const signal = Utils.useCancellation()
+  const stateRefreshTimer = useRef()
+
+  /*
+   * Data fetchers
+   */
+
+  Utils.useOnMount(() => {
+    const loadWorkflow = async () => {
+      const includeKey = [
+        'end', 'executionStatus', 'failures', 'start', 'status', 'submittedFiles:workflow', 'workflowLog', 'workflowName'
+      ]
+      const wf = await Ajax(signal).Workspaces.workspace(namespace, name).submission(submissionId).getWorkflow(workflowId, includeKey)
+      setWorkflow(wf)
+
+      if (_.includes(wf.status, ['Running', 'Submitted'])) {
+        stateRefreshTimer.current = setTimeout(loadWorkflow, 60000)
+      }
+    }
+
+    loadWorkflow()
+    return () => {
+      clearTimeout(stateRefreshTimer.current)
+    }
+  })
+
+  /*
+   * Page render
+   */
+  const { end, failures, start, status, workflowLog, workflowName, submittedFiles: { workflow: wdl } = {} } = workflow
+
+  const restructureFailures = failuresArray => {
+    const filtered = _.filter(({ message }) => !_.isEmpty(message) && !message.startsWith('Will not start job'), failuresArray)
+    const sizeDiff = failuresArray.length - filtered.length
+    const newMessage = sizeDiff > 0 ? [{ message: `${sizeDiff} jobs were queued in Cromwell but never sent to the cloud backend due to failures elsewhere in the workflow` }] : []
```

Comment:
Just for line length:
```suggestion
    const newMessage = sizeDiff > 0 ? [{
      message: `${sizeDiff} jobs were queued in Cromwell but never sent to the cloud backend due to failures elsewhere in the workflow`
    }] : []
```